Bottom: 89165a8573ed2ff38b41e067b12c1701f4afe6e9
Top:    a88b2cf277264a053dbfa24c3e6616b1b19e2f6d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:33:45 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/Vagrantfile b/Vagrantfile
index 3e886539..e037c694 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -1,6 +1,40 @@
+$bbworker = <<-SCRIPT
+yum install -y \
+    epel-release
+
+yum install -y \
+    device-mapper-persistent-data \
+    git \
+    lvm2 \
+    python-pip \
+    yum-utils
+
+yum-config-manager \
+  --add-repo \
+  https://download.docker.com/linux/centos/docker-ce.repo
+
+yum remove -y \
+    docker \
+    docker-client \
+    docker-client-latest \
+    docker-common \
+    docker-latest \
+    docker-latest-logrotate \
+    docker-logrotate
+
+yum install -y \
+    docker-ce
+
+pip install \
+    docker
+
+systemctl start docker
+docker run --rm hello-world
+SCRIPT
+
 Vagrant.configure("2") do |config|
 
-    config.vm.define "vmdaedalus.novatech-llc.com", primary: true do |main|
+    config.vm.define "vmdaedalus", primary: true do |main|
         main.vm.box = "daedalus"
         main.vm.box_url = "vmdaedalus/vmdaedalus-virtualbox.box"
         main.vm.hostname = "vmdaedalus.novatech-llc.com"
@@ -19,11 +53,15 @@ Vagrant.configure("2") do |config|
             mount_options: ["ro"]
     end
 
-    config.vm.define "buildbot-worker.novatech-llc.com" do |worker|
-        worker.vm.box = "centos/atomic-host"
-        worker.vm.hostname = "buildbot-worker.novatech-llc.com"
+    config.vm.define "bbworker" do |worker|
+        worker.vm.box = "centos/7"
+        worker.vm.hostname = "bbworker.novatech-llc.com"
         worker.vm.provider "virtualbox" do |vb|
             vb.name = "bbworker"
+            vb.cpus = 1
         end
+        worker.vm.network "private_network", ip: "192.168.0.150", netmask:"255.255.0.0"
+        worker.vm.provision "shell", inline: $bbworker
+
     end
 end
diff --git a/provision.sh b/provision.sh
index c3c88721..ad58dd05 100755
--- a/provision.sh
+++ b/provision.sh
@@ -9,7 +9,6 @@ ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o Contr
 exec ansible-playbook \
         --connection=ssh \
         --timeout=30 \
-        --limit="vmdaedalus.novatech-llc.com" \
         --inventory-file=/home/coopera/Developer/testdaedalus/vmdaedalus/vagrant_ansible_inventory \
-        -v \
+        -vv \
         ansible-playbook/site.yml "$@"
diff --git a/vmdaedalus/vagrant_ansible_inventory b/vmdaedalus/vagrant_ansible_inventory
index c6b5207d..9e73a23a 100644
--- a/vmdaedalus/vagrant_ansible_inventory
+++ b/vmdaedalus/vagrant_ansible_inventory
@@ -1,14 +1,17 @@
 # Generated by Vagrant
 
-vmdaedalus.novatech-llc.com network_connection="System enp0s8" ansible_host=127.0.0.1 ansible_port=2222 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/vmdaedalus.novatech-llc.com/virtualbox/private_key'
-buildbot-worker.novatech-llc.com ansible_host=127.0.0.1 ansible_port=2200 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/buildbot-worker.novatech-llc.com/virtualbox/private_key'
+vmdaedalus.novatech-llc.com ansible_host=127.0.0.1 ansible_port=2222 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/vmdaedalus/virtualbox/private_key'
+bbworker ansible_host=127.0.0.1 ansible_port=2200 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
 
 [bacula]
 vmdaedalus.novatech-llc.com
 
-[docker-hosts]
+[daedalus]
 vmdaedalus.novatech-llc.com
 
-[buildbot]
+[docker-hosts]
 vmdaedalus.novatech-llc.com
-buildbot-worker.novatech-llc.com
+bbworker
+
+[buildbot-worker]
+bbworker
